L1-CPP5term-Calculator/PluginsLoader.h

```diff
@@ -0,0 +1,20 @@
+#ifndef UNTITLED_PLUGINSLOADER_H
+#define UNTITLED_PLUGINSLOADER_H
+#include <Windows.h>
+#include <string>
+#include <map>
+#include <iostream>
+
+using unary = double(*)(double);

```

Comment:
No C++ here, the goal of various math funcs is to design a class that can store all necessary stuff about it. + That can help you design functions with 1, 2 and more parameters